src/client/components/RetriveCVs/RetriveCvs.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import { CardCvViewSearchComponent } from '../CardCvViewSearchComponent/CardCvViewSearchComponent';
+
+function RetriveCvs() {
+  const [cvs, setCvs] = useState([]);
+  useEffect(() => {
+    (async () => {
+      const result = await fetch('/api/cv').then((res) => res.json());
+      console.log(result);
+      setCvs(result);
+    })();
+  }, []);
+
+  return (
+    <div>
+      {cvs?.map((item) => (
+        <div>
```

Comment:
You can remove this <div> ;)

Reply:
okey, thanks i will do it,.i am first making a new endpoint to fetch cvsnUsers so that i don't use Kamal's one,.once it works,.i work on the frontend,.,.